waterlog/src/Components/LeakLitres.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchLeakLitres } from '../actions/LeakLitresActions';
+
+class LeakLitres extends Component {
+
+  componentDidMount() {
+    this.props.fetchLeakLitres();
+
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {}
+  }
+
+  render() {
+    const leakItem = this.props.litres;
+
+    return (
+      <div>
+        <h1>{leakItem.item.Item1}</h1>
+      </div>
+    )
+  }
+}
+//  {leakItem.item.Item1}
```

Comment:
Just remove Ghost code and remove empty spaces like line 10 and 37.
Please add an empty line at the end of this file